IPython/html/base/handlers.py

```diff
@@ -138,6 +138,10 @@ def base_url(self):
         return self.settings.get('base_url', '/')
 
     @property
+    def default_url(self):
+        return self.settings.get('default_url', '')
```

Comment:
Ah, I see, we're accessing `settings` directly in the redirect rather than using this property. Don't mind me.